TAF_EasyRest/TestFramework/PageComponents/Owner/CreateMenuPageComponents/CreateListMenuAndComponent.cs

```diff
@@ -0,0 +1,30 @@
+﻿
+namespace TestFramework.PageComponents.Owner.CreateMenuPageComponents
+{
+    public class CreateListMenuAndComponent
```

Comment:
The name of the class must end with ...PageComponent - this is our template. Please rename all your classes for page components.